From 1c28cac096183508b795a82c749c9323a6c76f03 Mon Sep 17 00:00:00 2001
From: icg_intg <icg_intg@intel.com>
Date: Mon, 23 Jul 2018 10:14:55 +0800
Subject: [PATCH 17/54] media: intel-ipu4: css: scci_IPU4_master_20180721_1100
 fw release

Message for Open Source:
New firmware css release.

Message for Internal:
[Feature]New firmware css release.
[Root Cause/Changes]
FW_Manifest_ID: 511354d834764ce69dd23f5a4a3629b0dabac0de
FW_Build_Branch: IPU4_master
FW_Build_Topic: auto_merge_queue
library version: commit-id 6fbdd19
FW Bundle patch number of this release :  343634
FW tag : scci_IPU4_master_20180721_1100

Change-Id: Id818c2d6c26e3ea467b2fa553dd95aa8effb102e
Depends-on: I6498fc0a5e8803ad0987dc984278d25b32b03ebf
Depends-on: Ia940da4f879305aea822a0abfc4ee07d0accef7a
Signed-off-by: icg_intg <icg_intg@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h           | 2 +-
 .../lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
index 9a9abe1..88ae90d 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20180709
+#define IA_CSS_FW_PKG_RELEASE  0x20180721
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
index 9a9abe1..88ae90d 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20180709
+#define IA_CSS_FW_PKG_RELEASE  0x20180721
-- 
1.9.1

